app/scripts/views/mixins/resend-mixin.js

```diff
@@ -43,8 +43,8 @@ define([
 
     _updateResendButton: function () {
       var self = this;
-      // Hide the button after 4 attempts. Redisplay button after a delay.
-      if (self._attemptedSubmits === TOO_MANY_ATTEMPTS) {
+      // Hide the button after too many attempts. Redisplay button after a delay.
+      if (self._attemptedSubmits >= Constants.EMAIL_RESEND_MAX_TRIES) {
         self.logScreenEvent('too_many_attempts');
```

Comment:
Hmm...  This alters the original behavior. What will happen is the first attempt will be submitted, the 2nd and 3rd are not, and on the 3rd attempt the button will be hidden. The `this._attemptedSubmits === 4` branch of https://github.com/mozilla/fxa-content-server/pull/2553/files#diff-2d294c0ba3ed94c511be9bc834c315f1R31 will never be hit.

I'm trying to think of a better solution here. One of the main problems is that the rate limiting code exists here and in fxa-client. It would be much easier to reason about if the logic lived in one place. I'm undecided whether that one place should be fxa-client, resend-mixin, or some third place. If the logic were in resend-mixin, then fxa-client would be stateless and could be created on demand instead of being passed around. If the logic were in fxa-client, then resend mixin could stop worrying about keeping track of submit count and instead only react to responses from the fxa-client. But neither of these seem exactly right, because this is business logic that should belong in a model of some sort. :/